Dockerfile

```diff
@@ -5,20 +5,18 @@ ENV CEREBRO_VERSION=0.6.6 \
 
 WORKDIR /opt/cerebro
 RUN set -ex;\
-  addgroup -S cerebro && adduser -S -G cerebro cerebro; \
```

Comment:
seems reasonable, but this sort of thing is all over docker library Dockerfiles, e.g. https://github.com/docker-library/elasticsearch/blob/master/Dockerfile-alpine.template#L4 is there a reason they are creating a group too?

Thinking a little more about this I don't see why we don't just run as `nobody`  and chown as `nobody:nogroup` you wouldn't want that on a multi user system, but a container isn't really that anyway...

Reply:
I think they want to run it as a user so they create a user. I love your `nobody` idea, the user and group are already there, so why not use it? I see a lot of people using it as well and it can be our standard for everything.